src/services/smartWallet.js

```diff
@@ -293,13 +294,15 @@ class SmartWallet {
     return items;
   }
 
-  getDeployEstimate(gasPrice: BigNumber) {
-    /**
-     * can also call `this.sdk.estimateAccountDeployment(REGULAR);`,
-     * but it needs sdk init and when migrating we don't have SDK initiated yet
-     * so we're using calculation method below that is provided by SDK creators
-     */
-    return utils.bigNumberify(790000).mul(gasPrice);
+  async estimateAccountDeployment(gasInfo: GasInfo) {
+    const deployEstimate = await this.sdk.estimateAccountDeployment().catch(() => {});
+    const gasAmount = get(deployEstimate, 'gasFee', 790000);
```

Comment:
`const gasAmount = get(deployEstimate, 'gasFee', new BigNumber(790000));`
to keep consistency

Reply:
Updated, please resolve if OK.